### ISSUE TITLE
Day 5 coal

### ISSUE THREAD

**Issue by Tzrt** [Zamiell/stardew-valley#38]
Should add a mention of bringing 1 coal home with you when leaving the mountain lake on day 5, for the scarecrow on day 6.

**Comment by Zamiell** [comment]
You usually have it already from breaking rocks on the farm, right?

**Comment by Tzrt** [comment]
Yes, but we bring our coal with us at the beginning of day 5 to the lake. The guide could mention to either leave 1 coal behind at home chest in the morning, or to bring a coal back with you at the end of day 5. I ended up crafting the scarecrow at the end of day 6 instead of the morning when I realized I all my coal was at the mountain lake with my furnaces.

**Comment by Zamiell** [comment]
can you submit a PR?

**Comment by djbucci** [comment]
I just ran into this as well. If other haven’t done so yet I can submit a PR. Just don’t want to step on anyone else’s.

**Comment by Zamiell** [comment]
no PRs are currently open (https://github.com/Zamiell/stardew-valley/pulls)

**Comment by Zamiell** [comment]
should be addressed now
